packages/actor-query-operation-leftjoin/test/ActorQueryOperationLeftJoin-test.ts

```diff
@@ -1,16 +1,21 @@
 import { BindingsFactory } from '@comunica/bindings-factory';
 import { ActorQueryOperation } from '@comunica/bus-query-operation';
 import { ActionContext, Bus } from '@comunica/core';
+import { QueryEngine } from '@comunica/query-sparql-rdfjs';
 import type { IQueryOperationResultBindings, Bindings, IJoinEntry } from '@comunica/types';
 import { ArrayIterator, UnionIterator } from 'asynciterator';
+import * as chai from 'chai';
+import * as n3 from 'n3';
 import { DataFactory } from 'rdf-data-factory';
 import * as sparqlee from 'sparqlee';
 import { isExpressionError } from 'sparqlee';
-import { ActorQueryOperationLeftJoin } from '../lib/ActorQueryOperationLeftJoin';
+import { ActorQueryOperationLeftJoin } from '../lib';
 import '@comunica/jest';
+import 'mocha';
 
 const DF = new DataFactory();
 const BF = new BindingsFactory();
+const sparql = new QueryEngine();
```

Comment:
Since this is a unit test specific to this actor, we should not import components from other actors.

Reply:
fixed.